<ui version="4.0" >
 <class>OrderDialog</class>
 <widget class="QDialog" name="OrderDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <property name="horizontalSpacing" >
    <number>6</number>
   </property>
   <property name="verticalSpacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="ordertab" >
      <attribute name="title" >
       <string>Order</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="label_10" >
           <property name="text" >
            <string>Client number:      </string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>101</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="clientlineEdit" />
         </item>
        </layout>
       </item>
       <item row="1" column="0" >
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>Phone:*   </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="phonelineEdit" >
           <property name="minimumSize" >
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="blacklisttoolButton" >
           <property name="focusPolicy" >
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="toolTip" >
            <string>Move to blacklist</string>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="icon" >
            <iconset resource="../icons/resources.qrc" >:/32/file_del.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4" >
           <property name="text" >
            <string>Order time:*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="orderdateTimeEdit" >
           <property name="displayFormat" >
            <string>dd.MM.yyyy H:mm</string>
           </property>
           <property name="calendarPopup" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0" >
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_3" >
           <property name="text" >
            <string>Address:*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="addresslineEdit" />
         </item>
        </layout>
       </item>
       <item row="3" column="0" >
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_9" >
           <property name="text" >
            <string>Go to:*    </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="gotolineEdit" />
         </item>
        </layout>
       </item>
       <item row="4" column="0" >
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>Type:       </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="parkingcomboBox" >
           <property name="minimumSize" >
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5" >
           <property name="text" >
            <string>Cost:*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="costlineEdit" >
           <property name="readOnly" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0" >
        <widget class="QCheckBox" name="deferredcheckBox" >
         <property name="layoutDirection" >
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text" >
          <string>Deferred</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>475</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Status" >
      <attribute name="title" >
       <string>Status</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>9</number>
       </property>
       <property name="topMargin" >
        <number>9</number>
       </property>
       <property name="rightMargin" >
        <number>9</number>
       </property>
       <property name="bottomMargin" >
        <number>9</number>
       </property>
       <property name="horizontalSpacing" >
        <number>6</number>
       </property>
       <property name="verticalSpacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_6" >
           <property name="text" >
            <string>Driver:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="drivercomboBox" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0" >
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_8" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Car:    </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="carlineEdit" >
           <property name="focusPolicy" >
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="readOnly" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0" >
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_7" >
           <property name="text" >
            <string>Real cost:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum" >
            <number>10000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cachecheckBox" >
           <property name="text" >
            <string>Cache</string>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="3" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" >
        <layout class="QGridLayout" >
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <property name="horizontalSpacing" >
          <number>6</number>
         </property>
         <property name="verticalSpacing" >
          <number>6</number>
         </property>
         <item row="0" column="1" >
          <widget class="QDateTimeEdit" name="waitdateTimeEdit" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="readOnly" >
            <bool>true</bool>
           </property>
           <property name="displayFormat" >
            <string>dd.MM.yyyy H:mm</string>
           </property>
           <property name="calendarPopup" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0" >
          <widget class="QCheckBox" name="waitcheckBox" >
           <property name="text" >
            <string>Wait client</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QCheckBox" name="freecheckBox" >
           <property name="text" >
            <string>Driver free</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QDateTimeEdit" name="godateTimeEdit" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="readOnly" >
            <bool>true</bool>
           </property>
           <property name="displayFormat" >
            <string>dd.MM.yyyy H:mm</string>
           </property>
           <property name="calendarPopup" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QDateTimeEdit" name="freedateTimeEdit" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="readOnly" >
            <bool>true</bool>
           </property>
           <property name="displayFormat" >
            <string>dd.MM.yyyy H:mm</string>
           </property>
           <property name="calendarPopup" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QCheckBox" name="gocheckBox" >
           <property name="text" >
            <string>Driver go</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="closepushButton" >
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text" >
        <string>Close</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="savepushButton" >
       <property name="text" >
        <string>Save</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelpushButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>clientlineEdit</tabstop>
  <tabstop>phonelineEdit</tabstop>
  <tabstop>orderdateTimeEdit</tabstop>
  <tabstop>addresslineEdit</tabstop>
  <tabstop>gotolineEdit</tabstop>
  <tabstop>parkingcomboBox</tabstop>
  <tabstop>costlineEdit</tabstop>
  <tabstop>deferredcheckBox</tabstop>
  <tabstop>drivercomboBox</tabstop>
  <tabstop>waitcheckBox</tabstop>
  <tabstop>waitdateTimeEdit</tabstop>
  <tabstop>gocheckBox</tabstop>
  <tabstop>godateTimeEdit</tabstop>
  <tabstop>freecheckBox</tabstop>
  <tabstop>freedateTimeEdit</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>cachecheckBox</tabstop>
  <tabstop>savepushButton</tabstop>
  <tabstop>cancelpushButton</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources>
  <include location="../icons/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>cancelpushButton</sender>
   <signal>clicked()</signal>
   <receiver>OrderDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>508</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel" >
     <x>494</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>waitcheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>waitdateTimeEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>66</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel" >
     <x>120</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gocheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>godateTimeEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>44</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel" >
     <x>133</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>freecheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>freedateTimeEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>45</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel" >
     <x>127</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
